<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signUpPage</class>
 <widget class="QWidget" name="signUpPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>姓名</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>学号</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>电话号码</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>密码</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>确认密码</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLineEdit" name="lineEdit">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="placeholderText">
              <string>请输入姓名</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_2">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="placeholderText">
              <string>请输入学号</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_3">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="placeholderText">
              <string>请输入手机号码</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_4">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>数字或字母 6到16位</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_5">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>再次输入密码</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>显示密码</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>注册</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
